src/content/docs/en/guides/upgrade-to/v6.mdx

```diff
@@ -105,11 +105,11 @@ The following deprecated features are no longer supported and are no longer docu
 
 Some deprecated features may temporarily continue to function until they are completely removed. Others may silently have no effect, or throw an error prompting you to update your code.
 
-### `Astro` in `getStaticPaths()`
+### Deprecated: `Astro` in `getStaticPaths()`
 
 <SourcePR number="14432" title="feat: deprecate Astro in getStaticPaths"/>
 
-In Astro 5.x, it was possible to access an `Astro` object inside `getStaticPaths()`. However, despite being typed the same as the `Astro` object accessible in the frontmatter, this object only had `site` and `generator` properties. This could lead to confusion about which `Astro` object properties were available inside `getStaticPaths()`.
+In Astro 5.0, it was possible to access an `Astro` object inside `getStaticPaths()`. However, despite being typed the same as the `Astro` object accessible in the frontmatter, this object only had `site` and `generator` properties. This could lead to confusion about which `Astro` object properties were available inside `getStaticPaths()`.
```

Comment:
```suggestion
In Astro 5.x, it was possible to access an `Astro` object inside `getStaticPaths()`. However, despite being typed the same as the `Astro` object accessible in the frontmatter, this object only had `site` and `generator` properties. This could lead to confusion about which `Astro` object properties were available inside `getStaticPaths()`.
```